### PR TITLE
#165 - sort now doesn't rerender but moves elements - much faster. Al…

### PR DESCRIPTION
**Pull request by kentmw**
…so fixed misleading teset cases for speed concerning list view update method (collection.reset). Also fixed dependency issues with adding NestedCell as viewState

### CODE REVIEW

File: modules/ListView.js [hunk 1]
```diff
@@ -430,7 +464,8 @@
      */
     __generateChildArgs: function(model) {
       var args = {
-        'context': _.extend({}, _.result(this, '__childContext'))
+        'context': _.extend({}, _.result(this, '__childContext')),
+        'listView': this
       };
```
Comment (mandragorn) [inline modules/ListView.js]: where is this used?

Comment (kentmw) [inline modules/ListView.js]: no where by torso. But if you want a child view to listen to events like "reorder" and other listview communication, i thought it couldn't hurt and could be very useful

File: modules/ListView.js [hunk 2]
```diff
@@ -430,7 +464,8 @@
      */
     __generateChildArgs: function(model) {
       var args = {
-        'context': _.extend({}, _.result(this, '__childContext'))
+        'context': _.extend({}, _.result(this, '__childContext')),
+        'listView': this
```
Comment (mandragorn) [inline modules/ListView.js]: Add this to the documentation somewhere.

**Comment by mandragorn** [comment]
+1 - just needs clear documentation on the parameters provided to a child view when it is created.

**Comment by mandragorn** [comment]
+2

**Pull request merged**